### DISCUSSION TITLE
unexpected food effect observed 24h after meal

### DISCUSSION THREAD

**Discussion by LetiziaCarrara** [Open-Systems-Pharmacology/Forum#767] [Questions & Problems] [answered]
Dear All, 

I am simulating food effect for a compound X. 

When food intake is simulated before drug administration, simulations show that also food taken 24 h before drug administration unexpectedly has an impact on drug PK.  

Based on what was reported  in issue  "Weibull for MealEffect TransitRate does not reach 1/GET after Meal " [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/552), I assumed that this could be due to the fact that the intestinal transit time at 24h is still lower than the value in fasted condition (verified with simulation).

Here-below my questions:
1) Is there something I am not considering in terms of assumptions and parameters to set?
2) if not, I am afraid that food effect might be overestimated.  Is there a way to overcome this issue, i.e., with some tricks or by modifying equations in MoBi? 

FYI, I am using OSPS version 7.1.0

Thank you very much in advance!
Best regards, 
Letizia

**Comment by PavelBal** [comment]
Dear Letizia,

unfortunately, this is a known problem of the current implementation of the food effect - the gastric emptying rate does not return to the fasted state after administration of food. I am also not aware of any solution.

**Comment by LetiziaCarrara** [comment]
Dear Pavel, 

thank you very much for your reply.  I'd like to know your opinion about the following possibilities:

1) to empirically (manually) change 1 or more parameter values in the food effect formula until the intestinal transit rate reaches the 1/GET value once the stomach is empty again.

2) to integrate the system "step by step" on different time intervals by using different simulations,  as follow
simulation 1: [time_food_administration  time*] --> in this time interval the transit rate in presence of food will be used  
simulation 2: [time* time_end] --> in this time interval a transit rate equal to 1/GET will be used, and the initial condition of each state will be updated to the value assumed by the corresponding state of the system at the end of the previous simulation. 
Of course, depending on the problem, more than 2 time intervals might be needed. 

Thanks again, 
Letizia

**Comment by arketkar** [comment]
@PavelBal Can there be an option of setting up 'fasted state' as an 'event' in PK-Sim?

**Comment by PavelBal** [comment, marked as answer]
@arketkar This would certainly be a workaround, but is not possible at the moment. Maybe you can create a feature request for in in [PK-Sim repository](https://github.com/Open-Systems-Pharmacology/PK-Sim)?

I can think of following workaround, which only works in MoBi.
The food effect in MoBi is defined in the "Events"-building block. The rate of gastric emptying is defined by the parameter `Organism|Lumen|Stomach|Intestinal transit rate (absolute)`. Upon meal administration, the value of this parameter is changed to a new formula:
![image](https://user-images.githubusercontent.com/25903040/87140872-77e1d200-c2a2-11ea-854d-0e0cc4161a89.png)

The value is reset when when condition `StomachIntestinalTransitRate > 1/GET` is met:
![image](https://user-images.githubusercontent.com/25903040/87140937-9051ec80-c2a2-11ea-9611-1d8738d48c64.png)

Unfortunately, the value of the food effect formula for `Intestinal transit rate (absolute)` does not return to the "fasted" state quickly enough, so the condition is not met even after 24h. You could modify the condition for the `Meal_StopEvent` from

`StomachIntestinalTransitRate > 1/GET`

to something like 

`(StomachIntestinalTransitRate > 1/GET) OR (Time >= 12*60)`

so simply assuming that after 12 hours (of simulation!) the gastric emptying is definitely at its fasted state. If you plan to simulate multiple food administrations, make sure that the `Meal_StopEvent` is always before the administration of the next meal.

This is how it would look like in the simulation. This a simulation of po caffeine administration with a food effect at t = 0. With the default gastric emptying event, the rate of gastric emptying (blue dashed line) remains low even after 24 hours post meal. WIth the adjusted event (pink dashed line), the rate returns to a fasted state after 12h. The concentration curves for caffeine remain identical. Keep in mind that if another meal would be administered at e.g. 11.5h, it would be "reset" half an hour later.
![image](https://user-images.githubusercontent.com/25903040/87141954-19b5ee80-c2a4-11ea-9f0f-b06d9e162a0a.png)

**Comment by LetiziaCarrara** [comment]
@PavelBal Thank you very much for your suggestion .

I just ask for a clarification about it: why resetting the gastric emptying rate after 12 hours? 
This is a crucial parameter that impacts the duration of food effect I am predicting.
Since in feed condition gastric emptying time is around 3-4 hours (with some variability depending on food type and quantity), does resetting the gastric emptying rate to the fasten state value after 3-4 hours make sense?

Thank you very much in advance, 
Letizia

**Comment by PavelBal** [comment]
@LetiziaCarrara 12h was just an example as I did not have any reliable values at hand. 4 hours indeed makes more sense.

**Comment by LetiziaCarrara** [comment]
@PavelBal thank you very much.
Letizia

**Comment by LetiziaCarrara** [comment]
@PavelBal  sorry if I bother you again. The solution you have suggested works, but it is quite steep. Having a smoother profile would be great.

I obtained it by changing the GET_alpha from the default value of 792 to 147 (I obtained 147 by implementing the formula in Matlab, but I may be wrong -a dimensional check was not easy to be performed-). This applies when GET=15 min (default value).

![food_2](https://user-images.githubusercontent.com/68049496/87776239-bf7ad780-c827-11ea-9002-bc19bb4edc7f.PNG)

![food_1](https://user-images.githubusercontent.com/68049496/87775329-344d1200-c826-11ea-854f-86b9abd43aff.PNG)

In my model I increased GET to 35.5 min [reference](https://link.springer.com/content/pdf/10.1186/1471-230X-11-145.pdf). When GET_alpha = 147 the gastric emptying rate returns to the fasted state after 30 min, which seems too short to me.  A more physiologically return to the fasted state after 4 hours was obtained by multiplying 147 by the ratio  between the two different GET, i.e. 147*(35.5/15).  

What do you think? 

Thank you very much, 
Letizia

**Comment by StephanSchaller** [comment]
If the 4h are as expected it would make sense. Be aware, that the meal effect is now static and will not change if a different meal (energy content and fraction solid) is defined.
Ideally, the constants in the GET_alpha/beta formula should be defined as variables that the user could adjust in order to customize the meal effect formula.

But as @PavelBal mentioned, this is certainly possible in MoBi.

**Comment by LetiziaCarrara** [comment]
thank you very much @StephanSchaller
